uabot.code-snippets

```diff
@@ -0,0 +1,741 @@
+			"\t\"arguments\": {\r",
+			"\t\t\"originLevel1\": \"${1:The new originLevel1}\",\r",
+			"\t\t\"originLevel2\": \"${2:Optional. The new originLevel2}\",\r",
+			"\t\t\"originLevel3\": \"${3:Optional. The new originLevel3}\"\r",
+			"\t}\r",
+			"},$0"
+		],
+		"description": "Coveo UABot SetOrigin Event"
+	},
+
+	"UABot Complete View Event": {
+		"prefix": "uaboteventview",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"View\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"clickRank\": ${1:The rank of the document to view (base: 0, random: -1)},\r",
+			"\t\t\"probability\": ${2:The probability that the user will view (between 0 and 1)},\r",
+			"\t\t\"pageViewField\": \"${3:The name of the field containing the page view unique identifier. Without @. Recommended: permanentid}\",\r",
+			"\t\t\"offset\": ${4:Optional. Offset used in random document viewing (integer)},\r",
+			"\t\t\"contentType\": \"${5:Optional. The type of content in the page.}\",\r",
+			"\t\t\"customData\": {\r",
+			"\t\t\t$0\r",
+			"\t\t}\r",
+			"\t}\r",
+			"},"
+		],
+		"description": "Coveo UABot Complete View Event"
+	},
+
+	"UABot Precise View Event": {
+		"prefix": "uaboteventview",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"View\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"clickRank\": ${1:The rank of the document to view (base: 0)},\r",
+			"\t\t\"probability\": ${2:The probability that the user will view (between 0 and 1)},\r",
+			"\t\t\"pageViewField\": \"permanentid\"\r",
+			"\t}\r",
+			"},$0"
+		],
+		"description": "Coveo UABot Precise View Event"
+	},
+
+	"UABot Random View Event": {
+		"prefix": "uaboteventview",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"View\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"clickRank\": -1,\r",
+			"\t\t\"probability\": ${1:The probability that the user will view (between 0 and 1)},\r",
+			"\t\t\"pageViewField\": \"permanentid\",\r",
+			"\t\t\"offset\": ${2:Offset used in random document viewing (integer)}\r",
+			"\t}\r",
+			"},"
+		],
+		"description": "Coveo UABot Random View Event"
+	},
+
+	"UABot SetReferrer Event": {
+		"prefix": "uaboteventsetreferrer",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"SetReferrer\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"referrer\": \"${1:The referrer URL}\"\r",
+			"\t}\r",
+			"},"
+		],
+		"description": "Coveo UABot SetReferrer Event"
+	},
+
+	"UABot Custom Event": {
+		"prefix": "uaboteventcustom",
+		"scope": "plaintext,json",
+		"body": [
+			"{\r",
+			"\t\"type\": \"Custom\",\r",
+			"\t\"arguments\": {\r",
+			"\t\t\"eventType\": \"${1:The event type of this custom event}\",\r",
+			"\t\t\"eventValue\": \"${2:The cause of this event, also the event value}\",\r",
+			"\t\t\"customData\": {\r",
+			"\t\t\t$0\r",
+			"\t\t}\r",
+			"\t}\r",
+			"},"
+		],
+		"description": "Coveo UABot Custom Event"
+	},
+
+	// Commerce events
```

Comment:
I would not include the specific commerce snippets here.

You could make a separate repository hosting your commerce scenarios and snippets if you wanted to remove those but still keep them for your team.